lib/matplotlib/widgets.py

```diff
@@ -417,17 +417,18 @@ def __init__(self, ax, label, valmin, valmax, *, valinit=0.5, valfmt=None,
         handle_style : dict
             Properties of the slider handle. Default values are
 
-            ========= ===== ======= ========================================
-            Key       Value Default Description
-            ========= ===== ======= ========================================
-            facecolor color 'white' The facecolor of the slider handle.
-            edgecolor color '.75'   The edgecolor of the slider handle.
-            size      int   10      The size of the slider handle in points.
-            ========= ===== ======= ========================================
-
-            Other values will be transformed as marker{foo} and passed to the
-            `~.Line2D` constructor. e.g. ``handle_style = {'style'='x'}`` will
-            result in ``markerstyle = 'x'``.
+            =============== ======= ========================================
+            Key             Default Description
+            =============== ======= ========================================
+            marker          'o'     The style of the slider handle.
+            facecolor       'white' The facecolor of the slider handle.
+            edgecolor       '.75'   The edgecolor of the slider handle.
+            size            10      The size of the slider handle in points.
+            =============== ======= ========================================
+
+            Please refer to:
+            https://matplotlib.org/stable/gallery/lines_bars_and_markers/marker_reference.html
```

Comment:
This should use a [Sphinx cross-reference](https://www.sphinx-doc.org/en/master/usage/referencing.html) instead of an absolute link.

Reply:
```suggestion
            :doc:`gallery/lines_bars_and_markers/marker_reference`
```
See https://matplotlib.org/devdocs/devel/document.html#refer-to-other-documents-and-sections